code/modules/mob/living/basic/farm_animals/chicken.dm

```diff
@@ -176,14 +177,15 @@ GLOBAL_VAR_INIT(chicken_count, 0)
 	if((. && prob(3) && eggsleft > 0) && egg_type)
 		visible_message("[src] [pick(layMessage)]")
 		eggsleft--
-		var/obj/item/E = new egg_type(get_turf(src))
+		var/obj/item/food/egg/E = new egg_type(get_turf(src))
 		E.scatter_atom()
 		if(eggsFertile)
-			if(GLOB.chicken_count < MAX_CHICKENS && prob(25))
+			if((GLOB.chicken_count <= GLOB.configuration.general.chicken_hatched_cap) && prob(25))
```

Comment:
This means you could technically bypass the cap, by having 50 chickens lay an egg, which all grow up into a total of 100 chickens. You probably should just not let eggs grow while theres too many chickens